base/server/python/pki/server/deployment/scriptlets/configuration.py

```diff
@@ -591,6 +591,12 @@ def spawn(self, deployer):
         if ocsp_uri:
             subsystem.config['ca.defaultOcspUri'] = ocsp_uri
 
+        ca_cert_path = deployer.mdict.get('pki_ca_cert_path')
+        destination = os.path.join(instance.nssdb_dir, "ca.crt")
+        if ca_cert_path and not os.path.exists(destination):
+            # TODO: determine correct uid/gid/permission parameters
+            pki.util.copyfile(ca_cert_path, destination)
```

Comment:
@edewata How do you suggest we do this? I wasn't able to find quickly where we provisioned the NSS DB and the permissions there. It'd be nice if there was an "inherit" mode that copied the permissions of the directory the target file is placed in.

Reply:
I think the NSS database is created here in the latest code:
https://github.com/dogtagpki/pki/blob/master/base/server/python/pki/server/deployment/scriptlets/security_databases.py#L86

You probably can call `instance.copyfile()` to create the file with the instance's permissions.

Reply:
Ah I see, `security_databases` executes prior to `configuration`, I gotcha now :)

Reply:
Ok, this should be resolved. Thanks @edewata!!